File: benchmarks/xlang/src/fib.py
import time
def fib(n):
    return n if n < 2 else fib(n - 1) + fib(n - 2)
t0 = time.perf_counter()
r = fib(32)
ms = int((time.perf_counter() - t0) * 1000)
print(f"CHECK {r}")
print(f"MS {ms}")

File: benchmarks/xlang/src/fib.rs
fn fib(n: i64) -> i64 { if n < 2 { n } else { fib(n - 1) + fib(n - 2) } }
fn main() {
    let t0 = std::time::Instant::now();
    let r = fib(32);
    println!("CHECK {}", r);
    println!("MS {}", t0.elapsed().as_millis());
}
